### PR TITLE
feat: Add maintainers-guide

### LINKED ISSUE #165: Feature: Update Maintainers Guide
### Type of feature

🍕 Feature

### Current behavior

I think we could better support maintainers with a more developed guide.

### Suggested solution

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by BekahHW**
## Description

 closes #165

- Creates a separate file for being a maintainer

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: docs/maintainers/maintainers-guide.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Join discord.
+- Prove your pizza worth!
+
+## What does the Maintainers Team do?
+Well, like all things in tech, it depends. We have different maintainer teams for each of the repositories, so it's important that you communicate with the admin maintainer if you have any questions.
+
+### Core Responsibilities
+Your responsibilities may include:
+- Reviewing and merging pull requests.
+- Providing comments and responses on PRs, issues, discussions, and Discord messages.
```
Comment (nickytonline) [inline docs/maintainers/maintainers-guide.md:26]: docs (blocking): explain the acronym before using it.

```suggestion
- Reviewing and merging pull requests (PRs).
- Providing comments and responses on PRs, issues, discussions, and Discord messages.
```

**Review by nickytonline — CHANGES_REQUESTED** [review]
This looks good to me. One small change request. Thanks for writing this up @BekahHW!

**Review by nickytonline — APPROVED** [review]
Thanks for making the change @BekahHW! 🚢

File: docs/maintainers/maintainers-guide.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+id: maintainers-guide
+title: "@open-sauced/maintainers-guide"
+sidebar_label: "@open-sauced/maintainers-guide"
+keywords:
+  - "@open-sauced/maintainers-guide"
+---
+
+Welcome to OpenSauced, Maintainers Guide for contributors who are interested in being part of the maintainer team for OpenSauced repositories.
```
Comment (CBID2) [inline docs/maintainers/maintainers-guide.md:9]: [proofreading] This is a great start @BekahHW! :) Just omit the parts I suggested below so the paragraph can be clearer. 
```suggestion
Welcome to OpenSauced! This guide is for contributors who are interested in being part of the maintainer team!  
```

File: docs/maintainers/maintainers-guide.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Welcome to OpenSauced, Maintainers Guide for contributors who are interested in being part of the maintainer team for OpenSauced repositories. 
+
+At OpenSauced, we're empowering contributors to work in the open and supporting maintainers to make data-driven decisions for their projects.
+
+It's important to us that we maintain a healthy environment for contributors and maintainers of our projects.
+
+## How do I Join the Maintainers Team?
+- Sign up for [opensauced.pizza](https://opensauced.pizza/)
+- Join discord.
+- Prove your pizza worth!
```
Comment (CBID2) [inline docs/maintainers/maintainers-guide.md:18]: [nitpick: While "prove your pizza worth" can be humorous, it implies that a person is not worthy to join if they don't contribute enough, which can turn some people away. It also goes against your goal to make more inclusive documentation. 
```suggestion
- Join our [Discord community](https://discord.com/invite/U2peSNf23P). 
- Be involved!
```

File: docs/maintainers/maintainers-guide.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+- Adhere to our [Code of Conduct](../contributing/code-of-conduct.md) and be an example for treating contributors with respect.
+- Communicate with the other maintainers in a timely and reasonable manner.
+- Understand and adhere to project standards.
+
+## Committing and Merging Changes
+1. For commit and pull request standards, please refer to [introduction to contributing](../contributing/introduction-to-contributing.md).
+2. Before merging in changes, be sure to run the project locally if UI changes were made.
+3. **Squash and merge commits** when you merge in a PR.
+
+Thank you for your interest in becoming a maintainer! Please reach out if you need help, guidance, or clarification at any time.
```
Comment (CBID2) [inline docs/maintainers/maintainers-guide.md:43]: [question/suggestion] Who can people contact if they are interested in being a maintainer? Consider adding this information to make the CTA clearer.

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @BekahHW! :) The guide is off to a great start! :) I just gave some suggestions and asked some questions to help you improve it.

**Pull request merged**